src/app-init/router.js

```diff
@@ -20,6 +20,7 @@ export const ROUTE_FRAGMENT_ADD = 'fragmentAdd';
 export const ROUTE_FRAGMENT_EDIT = 'fragmentEdit';
 export const ROUTE_FRAGMENT_DETAIL = 'fragmentDetail';
 export const ROUTE_DATA_MODEL_ADD = 'dataModelAdd';
+export const ROUTE_DATAMODEL_LIST = 'dataModelList';
```

Comment:
please add underscore: ROUTE_DATA_MODEL_LIST (as we did in ROUTE_DATA_MODEL_ADD )